src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -201,11 +203,13 @@ const SignOrExecuteForm = ({
           <ErrorMessage error={submitError}>Error submitting the transaction. Please try again.</ErrorMessage>
         )}
 
-        <Typography variant="body2" color="border.main" textAlign="center" mt={3}>
-          You&apos;re about to {txId ? '' : 'create and '}
-          {willExecute ? 'execute' : 'sign'} a transaction and will need to confirm it with your currently connected
-          wallet.
-        </Typography>
+        {safeTx && (
```

Comment:
This will make it jump as `safeTx` is async.

Reply:
I would just leave it even if the submit button is disabled.

Reply:
I would leave the `<AdvancedParams />` conditional however. We also have couple lines above the 
```
{tx && <DecodedTx tx={tx} txId={txId} />}
```
Wdyt? Otherwise if I render the `<AdvancedParams />` without the nonce, the Skeleton will be a bit misleading
<img width="605" alt="Screenshot 2022-10-05 at 14 00 21" src="https://user-images.githubusercontent.com/32431609/194055596-76f62d7d-f79e-41ed-b086-425f798632e6.png">

Reply:
It will make all other modals jump